drivers/include/sen5x.h

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright (C) 2023 TU Braunschweig Institut für Betriebssysteme und Rechnerverbund
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+ #ifndef SEN5X_H
+ #define SEN5X_H
```

Comment:
```suggestion
#ifndef SEN5X_H
#define SEN5X_H
```

The `#ifndef`s in your headers have a leading whitespace, which makes the `headerguards` static-test fail. Visual Studio Code likes to do that when inserting something after a block comment.